hibernate-rx-api/src/main/java/org/hibernate/rx/RxSession.java

```diff
@@ -187,7 +191,7 @@
 	 *
 	 * @see javax.persistence.EntityManager#createQuery(String)
 	 */
-	<R> RxQuery<R> createQuery(Class<R> resultType, String queryString);
+	<R> RxQuery<R> createQuery(String queryString, Class<R> resultType);
```

Comment:
Is this better? Not sure I think it is.

Reply:
It's consistent with `Session#createQuery`. Do you really think that one is better than the other? I don't mind to remove this change.

Reply:
Well I guess the argument for putting it second is that it's an optional argument, and optional arguments generally go after required args.

OTOH, since the HQL string is almost always going to be much *longer* than the class name, I feel like I would much prefer to put the class name first.

But I guess since both `Session` and `EntityManager` do it this way, we should be consistent.